docs/sidebar.json

```diff
@@ -89,5 +89,6 @@
   ],
   "Further Reading": [
     "further-reading/comparison"
+    "further-reading/kratos-video-tutorials"
```

Comment:
The link is not correct, the document is in `guides`

Reply:
Changed the documents location to further-reading.